### DISCUSSION TITLE
What strategy should I use for my proposal

### DISCUSSION THREAD

**Discussion by metrox-eth** [snapshot-labs/snapshot#752] [Q&A] [answered]
Suggest answers below or vote for an answer.

**Comment by metrox-eth** [comment, marked as answer]
It widely depends on your use case, if you want coin-voting (1 token= 1 vote) you can use "erc20-balance-of", however you may want:

- Delegate voting power using a delegation strategy
- Weighting voting power using a quadratic strategy
- NFT voting with an ERC-721 based strategy
- Only allow certain members to vote using whitelist strategy
- Calculate voting power from multiple chains with multichain strategy

You can combine up to 8 strategies on a single proposal (combination work as OR not AND opperator)

At the time of writing snapshot has over 150 voting strategies. Explore them here https://snapshot.org/#/strategies 
You can even preview actions using the playground button.

**Reply by DAC-Philantrop** [reply]
@metrox-eth 

> You can combine up to 8 strategies on a single proposal (combination work as OR not AND opperator)

Does that mean that if I have multiple strategies in my space, the voting power of an account will be calculated with the strategy that results in the highest voting power for that account or does it mean it will just add the voting power calculated by each strategy? 

I am trying to understand if there is a way to have multiple strategies and select one when creating a proposal, which will then be the only strategy used to calculate voting power for everyone on that proposal.

Appreciate any input.

**Reply by metrox-eth** [reply]
The voting power is cumulative. Each strategy add up.
